packages/api/src/mikro-orm/migrations/Migration20241119101706.ts

```diff
@@ -0,0 +1,7 @@
+import { Migration } from "@mikro-orm/migrations";
+
+export class Migration20241119101706 extends Migration {
+    async up(): Promise<void> {
+        this.addSql('alter table "BrevoConfig" add column "folderId" int not null;');
```

Comment:
If a brevoconfig has already been added here, it will crash. All existing data must be set to 1 here.

Reply:
a1a8d60be6929e0a8436c6a8dd1958601dc8afb5